helix-term/src/ui/menu.rs

```diff
@@ -77,7 +77,7 @@ impl<T: Item> Menu<T> {
         Self {
             options,
             editor_data,
-            matcher: Box::new(Matcher::default()),
+            matcher: Box::<fuzzy_matcher::skim::SkimMatcherV2>::default(),
```

Comment:
You don't need the turbo fish (type specification) here. `Bix::default()` should be enough and is much more readable

Reply:
Thanks so much @pascalkuthe for your review! 
Fixed

Reply:
You don't need to revert this changes entirely.
You reverted to `Box::new(Matcher::default())` but I mean that you can just use `Box::default()` here. 
With my comment I just ment that you don't need the `::<fuzzy_matcher::skim::SkimMatcherV2>` part

Reply:
Thanks for the clarification.
Fixed